sql/bis/_init.sql

```diff
@@ -62,14 +62,24 @@ OUTPUTS:
 	)
 */
 
+
 DROP TABLE IF EXISTS _INIT_BIS_devdb;
 WITH
 -- identify relevant_jobs
 JOBNUMBER_relevant as (
 	SELECT ogc_fid
 	FROM dob_jobapplications
 	WHERE jobdocnumber = '01'
-	AND jobtype ~* 'A1|DM|NB'
+	AND
+	( 
+		jobtype ~* 'A1|DM|NB' 
+	OR
+		(jobtype = 'A2' 
+		AND sprinkler is NULL 
+		AND lower(jobdescription) LIKE '%combin%' 
+		AND lower(jobdescription) NOT LIKE '%sprinkler%' 
+		AND lower(jobdescription) NOT LIKE '%commercial%')
+	)
 	AND gid = 1
 ) SELECT
```

Comment:
`jobdescription` will need to be included in the SELECT statement for Housing to review

Reply:
I'm not sure what you mean? This code is to put the correct A2 jobs into the `_INIT_devDB` table. All fields are pulled into the `UNITS_A2_devdb` table that housing will review, that code is in `sql/_units.sql`

Reply:
This code shouldn't affect which columns make it to the table we send to housing right? This is just a filter

Reply:
The `jobdescription` column from `dob_jobapplications` is added to the `_INIT_BIS_devdb` table as `job_desc`, that code is unchanged

Reply:
@td928 want to re-up this, think it's ok as is but want to make sure